.gitignore

```diff
@@ -29,10 +29,12 @@ yarn-error.log*
 
 # local env files
 .env*.local
+.env3
```

Comment:
**nitpick:** It should be `.env` not `.env3` - it will omit gitignore when someone adds his / hers own env file